web/frontend/tests/ballot.spec.ts

```diff
@@ -43,7 +50,74 @@ test('Assert ballot form is correctly handled for anonymous users, non-voter use
     await page.goto(`/ballot/show/${FORMID}`, { waitUntil: 'networkidle' });
     await expect(page).toHaveURL(`/ballot/show/${FORMID}`);
     await expect(castVoteButton).toBeVisible();
-    await expect(page.getByText(i18n.t('vote'))).toBeVisible();
+    await expect(page.getByText(i18n.t('vote'), { exact: true })).toBeVisible();
     await expect(page.getByText(i18n.t('voteExplanation'))).toBeVisible();
   });
 });
+
+test('Assert ballot is displayed properly', async ({ page }) => {
+  const content = await page.getByTestId('content');
+  // TODO integrate localisation
```

Comment:
Is this on the agenda? Or a "Far in the future" TODO?

Reply:
this is a reminder that this test is **not** taking into account localisation (there is a ticket for testing the localisation in general)

so it is a "if/when you implement the tests for the localisation, do not forget to adapt this one" as I took care to make all the other tests are localisation-independent